src/style.css

```diff
@@ -1,3 +1,82 @@
-body {
+* {
+  list-style: none;
+  text-decoration: none;
   margin: 0;
+  padding: 0;
+}
```

Comment:
Avoid setting values for `*` unless necessary - please move this to more specific selectors

For instance - move margin and padding to `html` or `body`, text decoration to link styles, ec